src/test/java/seedu/duke/logic/command/SellCommandTest.java

```diff
@@ -48,13 +44,43 @@ public void execute_itemWithInputIdExists_addsNormally() throws CommandException
         assertFalse(testList.contains("HarryPotter"));
         assertTrue(ShelfList.getShelfList().getShelf("soldItems").contains("HarryPotter"));
     }
+     */
 
     @Test
-    public void execute_itemWithInputIdDoesNotExist_throwsItemNotExistException()
-            throws CommandException, ShelfNotExistModelException,
-            IllegalArgumentModelException {
+    public void execute_itemWithInputIdDoesNotExist_throwsItemNotExistException() {
         itemID = "XXXXXXXX";
         testCommand = new SellCommand(itemID);
         assertThrows(ItemNotExistCommandException.class, () -> testCommand.execute());
```

Comment:
rename the function name exception thrown to the same one that is thrown when tested